src/Select.js

```diff
@@ -1048,7 +1051,8 @@ Select.propTypes = {
 	addLabelText: PropTypes.string,       // placeholder displayed when you want to add a label on a multi-value input
 	arrowRenderer: PropTypes.func,        // Create drop-down caret element
 	autoBlur: PropTypes.bool,             // automatically blur the component when an option is selected
-	autofocus: PropTypes.bool,            // autofocus the component on mount
+	autofocus: PropTypes.bool,            // [DEPRECATED] autofocus the component on mount
+	autoFocus: PropTypes.bool,						// autofocus the component on mount
```

Comment:
@gwyneplaine please fix the in-line indentation here, should be using spaces for consistency